lib/cylc/scheduler.py

```diff
@@ -1113,27 +1040,78 @@ def load_suiterc(self, is_reload=False):
             'CYLC_SUITE_NAME': self.suite,
             'CYLC_CYCLING_MODE': str(
                 self.config.cfg['scheduling']['cycling mode']),
-            'CYLC_SUITE_INITIAL_CYCLE_POINT': str(self.initial_point),
-            'CYLC_SUITE_FINAL_CYCLE_POINT': str(self.final_point),
+            'CYLC_SUITE_INITIAL_CYCLE_POINT': str(self.config.initial_point),
+            'CYLC_SUITE_FINAL_CYCLE_POINT': str(self.config.final_point),
         })
 
-    def _load_suite_params_1(self, _, row):
-        """Load previous initial cycle point or (warm) start cycle point.
-
-        For restart, these may be missing from "suite.rc", but was specified as
-        a command line argument on cold/warm start.
+    def _load_suite_params(self, row_idx, row):
+        """Load a row in the "suite_params" table in a restart.
 
+        This currently includes:
+        * Initial/Final cycle points.
+        * Start/Stop Cycle points.
+        * Suite UUID.
+        * A flag to indicate if the suite should be held or not.
         """
+        if row_idx == 0:
+            LOG.info('LOADING suite parameters')
         key, value = row
-        if key == 'initial_point':
-            self.cli_initial_point_string = value
-            self.task_events_mgr.pflag = True
-        elif key in ['start_point', 'warm_point']:
+        if key == 'icp':
```

Comment:
Will update to allow backward compat. In the new scheme, however, I'm going to modify the setting so that the calculated ICP will no longer be stored. (It is not used any way.)

Reply:
Done.